web/styles/rendered_markdown.css

```diff
@@ -595,6 +595,12 @@
         margin-top: -4px;
     }
 
+    /* Remove the outline when clicking on the copy-to-clipboard button
+    Fixing issue #25533 */
```

Comment:
This comment on the issue isn't needed, you could remove it.

Reply:
Thanks, updated.